exasol_script_languages_container_tool/lib/tasks/install_starter_scripts/run_starter_script_installation.py

```diff
@@ -0,0 +1,64 @@
+import os
+import shutil
+import sys
+from pathlib import Path
+import pkg_resources
```

Comment:
Use 
https://pypi.org/project/importlib-resources/ instead pkg_resource, importlib resources will is standard in 3.9, see more details here


https://docs.python.org/3/library/importlib.html#module-importlib.resources

Reply:
Besides the resources, I also need to get the current version of script-languages-container-tools.
This is is supported by  importlib_metadata, which came in to Python 3.8.
This would mean for getting the version I am stuck with pkg_resources and for getting the resources I could use the newer importlib_resources.

I would suggest to fix all together when we switch to Python 3.8.

Reply:
I think, we should fairly soon go to python 3.8, so maybe preparing as much as possible for it, might be not a bad idea.

> Use of pkg_resources is discouraged in favor of [importlib.resources](https://docs.python.org/3/library/importlib.html#module-importlib.resources), [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html), and their backports ([importlib_resources](https://pypi.org/project/importlib_resources), [importlib_metadata](https://pypi.org/project/importlib_metadata)). Please consider using those libraries instead of pkg_resources.

https://setuptools.pypa.io/en/latest/pkg_resources.html

Reply:
instead of `pkg_resources.get_distribution`, you can also use https://pypi.org/project/importlib-metadata/ which is compatible to `from importlib.metadata import version` from Python 3.8